txmanager.go

```diff
@@ -14,15 +16,28 @@ type TxManager interface {
 	WithTransaction(ctx context.Context, txfn TxFn) error
 }
 
+type TxManagerGormV2 interface {
```

Comment:
@aalexanderkevin I think we don't need this TxManagerGormV2, the signature is the same with TxManager. Wdyt?

Reply:
thanks mas done ya https://github.com/shortlyst-ai/go-txmanager/pull/3/commits/b95e120d6cd024e722de7cafffe5450a00e4aadf
@hendra-shortlyst